cypress/integration/currentTrip-user-flow.js

```diff
@@ -0,0 +1,59 @@
+import { aliasQuery, aliasMutation } from '../utils/graphql-test-utils';
+
+
+describe('Current Trip User Story', () => {
+
+beforeEach(() => {
+    cy.intercept('POST', 'https://walk-safe-backend.herokuapp.com/', (req) => {
+
+      aliasQuery(req, 'GetUser')
+
+      aliasMutation(req, 'CreateTrip')
+    })
+  })
+
+  describe('Should be able to start a new trip', () => {
+
+    it('Should be able to complete form and start a new trip', () => {
+      cy.visit('http://localhost:3000/')
+      cy.get('.start-point').type('2343 Quitman St Denver')
+          .wait(200)
+          .type('{downarrow}')
```

Comment:
Really cool solution, nice work figuring out the .type syntax to click/select different elements.

Reply:
^^^ I agree. The down arrow is a great addition